### ISSUE TITLE
Beg for a better experience of 'Vibration' on steamdeck win11！

### ISSUE THREAD

**Issue by Brent-Huifeng** [Valkirie/HandheldCompanion#1184]
The vibration is **continous** when playing GTA5 via **steamOS**, like the vibration of **iPhone**, while the vibration playing GTA5 in **win11** via **handheld companion** feels like raindrops hitting an iron plate **repeatedly**, making the experience feel **disjointed**. This kind of vibration always pulls me and my friends out of our favourite games. So I wonder if you can test the vibration difference between these 2 system, and try to produce a feature to adjust the vibration in win11 **continous**.
(Maybe you can make an '**input box**' where users can enter the '**vibration frequency**' 'times/s')
Thank you so much!!!

**Comment by Valkirie** [comment]
Hey, thanks for your feedback. Right now the delay between two rumble is 20ms. How "slow" would you say it's compared to steamOS experience ?

**Comment by Brent-Huifeng** [comment]
Yes yes, the delay of steamOS or other controller like NS PRO between 2 rumble is absolute smaller than 20ms, so I wonder if you can make this 'interval' adjustable to our users. It will be very helpful!!

**Comment by Valkirie** [comment]
We can make it anything we want. I'm not sure we should make it a customizable slider but rather guess the value from steamOS. Would you say it's twice, three or four times faster ?

**Comment by Brent-Huifeng** [comment]
The frequency of SteamOS is 100Hz-300Hz（ChatGpt） and iPhone is 130Hz-180Hz（Google）. So maybe we can try 150Hz, which is 6.6ms between 2 rumble（**3 times** compared to current Handheld Companion frequency）. Thank you!

![IMG_2924](https://github.com/user-attachments/assets/c6721a8b-ea13-4f47-ac84-27fb7c7ec17d)
![IMG_2925](https://github.com/user-attachments/assets/1e4597cb-a5f0-43fa-924a-fb97541e76ab)

**Comment by Valkirie** [comment]
You can install this beta build where I added an interval slider : https://1drv.ms/u/s!AleMgLLgxwGJiJpeEEtyEpa0K73h2Q?e=veLaTL. I honestly don't believe it's providing major improvements, even with a 1ms interval (1000Hz). I guess it's due to our unperfect rumble translation.

**Comment by Brent-Huifeng** [comment]
If further optimization is troublesome, just adding the **interval slider** to allow us to change the frequency to 1ms would be good enough!! Thank you!

**Comment by Brent-Huifeng** [comment]
Hi Valkirie, will the "interval slider" be added to a future version? I have been using the beta version for a few days and find I can't play without it.
